code/modules/mob/living/carbon/alien/humanoid/humanoid.dm

```diff
@@ -32,7 +32,7 @@
 /mob/living/carbon/alien/humanoid/attack_hulk(mob/living/carbon/human/user)
 	if(user.a_intent == "harm")
 		..(user, 1)
-		adjustBruteLoss(14 + rand(1,9))
+		adjustBruteLoss(15)

```

Comment:
I meant a real maintainer, not a bitch boy who carries their socks and tags pr's